RotaryWheelControl/RotaryWheel.xaml.cs

```diff
@@ -143,6 +145,52 @@ public RotaryWheel()
             };
         }
 
+        private void SpinTo(int itemIndex, int durationInSec = -1)
+        {
+            Random r = new Random(DateTime.Now.Millisecond);
+            var angleFromYAxis = 360 - Angle;
+            SelectedItem = _pieSlices
+                .SingleOrDefault(p => p.StartAngle <= angleFromYAxis && (p.StartAngle + p.Angle) > angleFromYAxis);
+
+            int count = _pieSlices.Count;
+            int currIndex = _pieSlices.IndexOf(SelectedItem);
+            int fullSpin = itemIndex / count;
+            itemIndex = itemIndex % count;
+            int steps = currIndex-itemIndex;
+            if (steps < 0)
+                steps = count + steps;
+
+            var startAngle = SelectedItem.StartAngle + SelectedItem.Angle / 2;
+            var finalAngle = startAngle + fullSpin*360 + steps*360/count;
+
+            doubleAnimation.From = startAngle;
+            doubleAnimation.To = finalAngle;
+            if(durationInSec>0)
+                doubleAnimation.Duration = new Windows.UI.Xaml.Duration(new TimeSpan(0, 0, durationInSec));
+            else
+                doubleAnimation.Duration = new Windows.UI.Xaml.Duration(new TimeSpan(0, 0, r.Next(3, 6)));
+            storyBoard.Begin();
+            storyBoard.Completed += StoryBoard_Completed;
+            Angle = ((int)finalAngle) % 360;
+
+        }
+        /// <summary>
+        /// Spins the wheel randomly.
+        /// </summary>
+        /// <param name="maxSpins">Maximum no. of spins or revolutions.</param>
+        /// <param name="durationInSec">Spin duration in Second. [-1 denotes random duration]</param>
+        public void Spin(int maxSpins=5, int durationInSec = -1)
+        {
+            Random r = new Random(DateTime.Now.Millisecond);
```

Comment:
Seed defines how the random numbers will be generated. As you are using `DateTime.Now.Milliseconds`. It's equivalent to using the default value (ie. passing in no argument).